mopidy_gmusic/library.py

```diff
@@ -495,16 +495,28 @@ def _to_mopidy_track(self, song):
         track_id = song.get('id', song.get('nid'))
         if track_id is None:
             raise ValueError
-        return Track(
-            uri='gmusic:track:' + track_id,
-            name=song['title'],
-            artists=[self._to_mopidy_artist(song)],
-            album=self._to_mopidy_album(song),
-            track_no=song.get('trackNumber', 1),
-            disc_no=song.get('discNumber', 1),
-            date=unicode(song.get('year', 0)),
-            length=int(song['durationMillis']),
-            bitrate=320)
+        try:
+            return Track(
+               uri='gmusic:track:' + track_id,
+               name=song['title'],
```

Comment:
This could be removed from the try-catch and replaced with song.get('title', 'Unknown')